Memerator/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+   "author": {
+      "name": "Chew",
+      "id": "476488167042580481"
+   },
+   "url": ["memerator.me", "beta.memerator.me"],
+   "description": {
+      "Memerator": "Memerator is a up and coming meme website where you can post and rate other's memes."
```

Comment:
The description is a keypair value that takes the language and the language description. Replace the "Memerator" key with "en"